test/testcafe/spec/ChallengeFactorEmail_MagicLink_spec.js

```diff
@@ -38,3 +49,25 @@ test
     await t.expect(pageTitle).eql('This email link has expired. To resend it, return to the screen where you requested it.');
   });
 
+  test
+  .requestHooks(logger, resendEmailMock, magicLinkEmailSentMock)
+  (`challenge email factor with magic link sent renders and has resend link`, async t => {
+    const challengeFactorPageObject = await setup(t);
+    await t.expect(challengeFactorPageObject.resendEmailView().getStyleProperty('display')).eql('none');
+    // wait for resend button to appear
+    await t.wait(65000);
+    // making sure we keep polling while we wait for the resend view to appear
+    await t.expect(logger.count(
```

Comment:
nit: 
- add assertion that logger has total count 17.
- explain a bit why 16 poll (Math.floor(65000/4000))